atom/renderer/atom_renderer_client.cc

```diff
@@ -121,20 +121,24 @@ bool IsDevToolsExtension(content::RenderFrame* render_frame) {
       .SchemeIs("chrome-extension");
 }
 
+std::vector<std::string> ParseSchemesCLISwitch(const char* switch_name) {
+  base::CommandLine* command_line = base::CommandLine::ForCurrentProcess();
+  std::string custom_schemes = command_line->GetSwitchValueASCII(switch_name);
+  if (!custom_schemes.empty()) {
```

Comment:
`base::SplitString` should handle empty strings so I don't think we need this check here.